index.html

```diff
@@ -0,0 +1,114 @@
+<html>
+  <head>
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <link id='yellowtail' href="https://fonts.googleapis.com/css?family=Yellowtail" rel="stylesheet">
+    <title>Chocolate Pizza</title>
+  </head>
+  <body>
+    <header id="header">
+      <div id="topleft">
+        <img class='' src="img/logo.png" alt=" Logo"/>
+        <section>
+          <h3 id="delicious">Delicious</h3>
+            <p id='bestFood'>THE BEST FOOD BLOG ON THE WEB.</p>
+        </section>
+          </div>
```

Comment:
watch your indentation here, this closing div should be at the same level as its corresponding opening tag.